src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -1017,16 +1017,15 @@ public function testColumnStyle()
 
     public function testThrowsWhenTheCellInAnArray()
     {
-        $table = new Table($this->getOutputStream());
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('A cell must be a TableCell, a scalar or an object implementing "__toString()", "array" given.');
+        $table = new Table($output = $this->getOutputStream());
         $table
             ->setHeaders(['ISBN', 'Title', 'Author', 'Price'])
             ->setRows([
                 ['99921-58-10-7', [], 'Dante Alighieri', '9.95'],
             ]);
 
-        $this->expectException(InvalidArgumentException::class);
-        $this->expectExceptionMessage('A cell must be a TableCell, a scalar or an object implementing "__toString()", "array" given.');
-
         $table->render();
```

Comment:
As the error is expected in `setRows`, this rendering is dead code.

Reply:
indeed

Reply:
should this removal be done in https://github.com/symfony/symfony/pull/52365 ?

Reply:
Will do 👌🏻

Reply:
Done